external/fv3fit/fv3fit/reservoir/readout.py

```diff
@@ -1,14 +1,10 @@
-import dacite
```

Comment:
The changes in this module:
- Simplify the readout class to be initialized with coefficients and intercepts from a trained regressor, and remove the `.fit` method. The fitting of the regressor will be done in the training function by calling `BatchLinearRegressor.batch_update`. 
- Save the readout coefficients as a sparse scipy array to save on space.
- Remove the `CombinedReservoirComputingReadout` class. Instead, a standalone `combine_readouts` function can be used to combine multiple readouts into a single block diagonal coefficients matrix. (Initially when I wrote this class I thought that subdomains' readouts would be trained and saved in parallel and we would then load individual readouts before combining. After writing the training function I realized this was less efficient and all readouts could be solved at once and combined without intermediate saves.)